src/java/org/apache/cassandra/locator/EndpointsForRange.java

```diff
@@ -40,13 +37,13 @@
 public class EndpointsForRange extends Endpoints<EndpointsForRange>
 {
     private final Range<Token> range;
-    private EndpointsForRange(Range<Token> range, List<Replica> list, boolean isSnapshot)
+    private EndpointsForRange(Range<Token> range, ReplicaList list)
```

Comment:
Unused?

Reply:
Looks like the intention was to make `byEndpoint` calculated lazily